doc/modules/kernel_approximation.rst

```diff
@@ -36,8 +36,47 @@ is advisable to compare results against exact kernel methods when possible.
 Nystroem Method for Kernel Approximation
 ----------------------------------------
 The Nystroem method, as implemented in :class:`Nystroem` is a general method
-for low-rank approximations of kernels. It achieves this by essentially subsampling
-the data on which the kernel is evaluated.
+for reduced rank approximations of kernels. It achieves this by subsampling without replacement
+rows/columns of the data on which the kernel is evaluated.
+Computational complexity for this method is :math:`\mathcal{O}(n^2_{\text{components}} \cdot n_{\text{samples}})` [WS2001]_
```

Comment:
Whereas the exact kernel calculation is is :math:`\mathcal{O}(n^3_{\text{samples}})`.
One can set :math:`n_{\text{components}} \ll n_{\text{samples}}` without a significative decrease in performance.